fenix/app/src/main/res/values/strings.xml

```diff
@@ -1160,6 +1160,8 @@
     <string name="share_copy_link_to_clipboard">Copy to clipboard</string>
     <!-- Toast shown after copying link to clipboard -->
     <string name="toast_copy_link_to_clipboard">Copied to clipboard</string>
+    <!-- Text for confirmation "snackbar" shown after copying an image to the clipboard. -->
+    <string name="snackbar_copy_image_to_clipboard_confirmation">Image copied to clipboard</string>
```

Comment:
Was this meant to be part of different change?

Reply:
Was part of the above change but no longer required with your suggestion 🙂